src/App.tsx

```diff
@@ -18,6 +21,30 @@ const StyledDiv = styled.div`
   height: 100vh;
 `;
 
+const ErrorHeader = styled.h2`
+  font-size: 1rem;
+  line-height: 1.5rem;
+  margin: 0px;
+  margin-bottom: 1em;
+`;
+
+const ErrorText = styled.p`
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  margin: 0px;
+`;
+
+const ErrorList = styled.ul`
+  padding-left: 1em;
+  margin: 0px;
+`;
+
+const ErrorItem = styled.li`
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  margin: 0px;
+`;
```

Comment:
I think this could probably be moved into `Modal.tsx` or into its own file, just to keep the main app file clutter-free.

Reply:
Can we actually just have the styles in the bottom of the files now?

Reply:
Maybe we should just have an `Error` component actually

Reply:
Added a FileErrorModal component to tackle this.